Model/LdapUser.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+namespace FR3D\LdapBundle\Model;
+
+use Symfony\Component\Security\Core\User\AdvancedUserInterface;
+
+/**
+ * Simple user object.
+ */
+class LdapUser implements LdapUserInterface, AdvancedUserInterface
+{
+    use UserRoleTrait;
+
+    const ROLE_DEFAULT = 'ROLE_USER';
+
+    /**
+     * @var string
+     */
+    protected $dn;
+
+    /**
+     * @var string
+     */
+    protected $username;
+
+    /**
+     * @var string
+     */
+    protected $email;
```

Comment:
email is not part of any implemented interface. Should we provide this?

Really I preffer if the user implements the final class with his custom needs instead of rely on magical general purpose class.